src/player.js

```diff
@@ -2036,7 +2036,7 @@ export default class Player extends FakeEventTarget {
     }
     this.ready()
       .then(() => {
-        if (this.isLive() && !this.isDvr()) {
+        if (this.isLive() && (!this.isDvr() || (this.isDvr() && typeof this.currentTime === 'number' && this.currentTime <= 0))) {
```

Comment:
`currentTime < 0` is invalid. the getter should return NaN
(also `this.isDvr()` here is redundant)

Reply:
current time could be negative cause we calculating it by currentTime of video Tag - getStartTimeOfDvrWindow() which means that if we have a DVR window of 2 hours and I'll seek to start and pause the DVR window is moving and current time become negative.
`isDvr` is important here cause There are two scenarios:
1. live without DVR which should always go to live edge(old one)
2. live DVR with negative currentTime(out of DVR window) (new one)

Reply:
You are correct `currentTime < 0` is invalid and it'll happen only on live DVR which will resolve here.